schematic/models/validate_manifest.py

```diff
@@ -244,10 +244,19 @@ def validate_manifest_values(self, manifest, jsonSchema
             for error in sorted(v.iter_errors(annotation), key=exceptions.relevance):
                 errorRow = i + 2
                 errorCol = error.path[-1] if len(error.path) > 0 else "Wrong schema"
+                errorColName = error.path[0] if len(error.path) > 0 else "Wrong schema"
                 errorMsg = error.message[0:500]
                 errorVal = error.instance if len(error.path) > 0 else "Wrong schema"
 
-                errors.append([errorRow, errorCol, errorMsg, errorVal])
+                errors.append([errorRow, errorColName, errorCol, errorMsg, errorVal])
```

Comment:
Take a look at how errors are added in the `ValidateAttribute` object: [ex](https://github.com/Sage-Bionetworks/schematic/blob/7df6950c248c90904bb7214b56cb25d7354f932b/schematic/models/validate_attribute.py#L443), to match style